lib/generators/provider/templates/Rakefile

```diff
@@ -1,8 +1,20 @@
+require 'bundler/setup'
+require 'bundler/gem_tasks'
+
 begin
-  require 'bundler/setup'
+  require 'rspec/core/rake_task'
+
+  APP_RAKEFILE = File.expand_path("../spec/manageiq/Rakefile", __FILE__)
+  load 'rails/tasks/engine.rake'
 rescue LoadError
-  puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
 end
 
-APP_RAKEFILE = File.expand_path("../spec/manageiq/Rakefile", __FILE__)
-load 'rails/tasks/engine.rake'
+namespace :spec do
+  desc "Setup environment for specs"
+  task :setup => 'app:test:providers:<%= provider_name %>:setup'
```

Comment:
I'm still not sold on the tasks living in `app:test:providers:` but I understand your reason